app.js

```diff
@@ -82,5 +83,16 @@ function encryptSentence(inputSentence) {
 }
 
 function letterCheck(clickedLetter) {
-  alert(clickedLetter);
+  state.roundCounter++;
+  const letterPositions = charPositions(clickedLetter, state.proverb);
+  let currentDisplayedProverb = state.displayedProverb;
+  letterPositions.forEach(item => currentDisplayedProverb = currentDisplayedProverb.substr(0, item) + clickedLetter + currentDisplayedProverb.substr(item + 1));
+  state.displayedProverb = currentDisplayedProverb;
+  sentence.innerHTML = state.displayedProverb;
+}
+
+function charPositions(letter, proverb){
+  var positions=[],i=-1;
```

Comment:
what is this? Why var? :O 
make it explicit
const positions = [];
let i = -1;